docs/modules/user-guide/partials/con_devfile.adoc

```diff
@@ -38,3 +38,4 @@ Developers use devfiles to focus only on developing their applications. With dev
 * Work with devfiles: xref:authoring-stacks.adoc[]
 * Update devfiles: xref:migrating-to-devfile-v2.adoc[]
 * Learn more about devfiles: xref:devfile-samples.adoc[]
+* Devfile registry: link:https://github.com/devfile/registry[devfile/registry]
```

Comment:
Should this be registry.devfile.io, and we also include a link to the devfile registry source repository (your link above)?

Reply:
true, thats the src. updated